libvast/src/defaults.cpp

```diff
@@ -18,7 +18,9 @@
 namespace vast::defaults::import {
 
 size_t test::seed(const caf::settings& options) {
-  if (auto val = caf::get_if<size_t>(&options, "vast.import.test.seed"))
+  if (auto val = caf::get_if<caf::config_value::integer>(&options, "vast."
+                                                                   "import."
+                                                                   "test.seed"))
```

Comment:
If you add a `//` after `&options,`, you force clang-format to break the line and not break the string. This is quite handy for config options, which we never want to break so this continues to shop up for `git grep vast.import.test.seed`.

The same applies to other instances in the same PR, sadly we have to force clang-format to do the right thing here. :|